### PR TITLE
NF: Comment String "reviews"

### PR DESCRIPTION
**Pull request by Arthur-Milchior**

### CODE REVIEW

**Review by david-allison-1 — APPROVED** [review]

File: AnkiDroid/src/main/res/values/07-cardbrowser.xml [hunk 1]
```diff
@@ -94,7 +94,7 @@
     <string name="card_info_due">Due</string>
     <string name="card_info_interval">Interval</string>
     <string name="card_info_ease">Ease</string>
-    <string name="card_info_reviews">Reviews</string>
+    <string name="card_info_reviews" comment="Number of time a card was reviewed">Reviews</string>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/values/07-cardbrowser.xml:97]:
```suggestion
    <string name="card_info_reviews" comment="Number of times a card was reviewed">Reviews</string>
```

File: AnkiDroid/src/main/res/values/10-preferences.xml [hunk 2]
```diff
@@ -197,7 +197,7 @@
         <item quantity="other">%1$s\n%2$d decks use this group</item>
     </plurals>
     <string name="deck_conf_new_cards">New cards</string>
-    <string name="deck_conf_rev_cards">Reviews</string>
+    <string name="deck_conf_rev_cards" comment="Name of the submenu of deck option allowing to configure reviews of due cards.">Reviews</string>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/values/10-preferences.xml:200]:
```suggestion
    <string name="deck_conf_rev_cards" comment="Name of the deck options submenu for configuring reviews of due cards.">Reviews</string>
```

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Actually, minor wording fixes

File: AnkiDroid/src/main/res/values/10-preferences.xml (hunk 2 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/res/values/10-preferences.xml:200]: done

**Review by david-allison-1 — APPROVED** [review]

**Pull request merged**